web-client/integration-tests/journey/docketClerkCreatesATrialSession.js

```diff
@@ -89,13 +89,12 @@ export const docketClerkCreatesATrialSession = (test, overrides = {}) => {
 
     expect(test.getState('validationErrors')).toEqual({
       startDate: errorMessages.startDate[1],
-      term: errorMessages.term,
```

Comment:
Same comment as above - I think we want to create an invalid term so we can prove a validation message is displayed.